### ISSUE TITLE
requireNativeComponent: "RCTWebView" was not found in the UIManager.

### ISSUE THREAD

**Issue by dbm5858** [microsoft/react-native-windows#7793]
Reported as fixed in #7758.  I'm still having the same problem.

## Environment
Run the following in your terminal and copy the results here.
1. `npx react-native --version`: 5.0.1-alpha.2
2. `npx react-native info`:
System:
    OS: Windows 10 10.0.18363
    CPU: (8) x64 Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz
    Memory: 6.56 GB / 15.86 GB
  Binaries:
    Node: 12.16.1 - C:\Program Files\nodejs\node.EXE
    Yarn: Not Found
    npm: 6.13.4 - C:\Program Files\nodejs\npm.CMD
    Watchman: Not Found
  SDKs:
    Android SDK:
      API Levels: 29, 30
      Build Tools: 28.0.3, 30.0.2
      System Images: android-30 | Google APIs Intel x86 Atom
      Android NDK: Not Found
    Windows SDK:
      AllowDevelopmentWithoutDevLicense: Enabled
      AllowAllTrustedApps: Enabled
      Versions: 10.0.18362.0, 10.0.19041.0
  IDEs:
    Android Studio: Version  4.1.0.0 AI-201.8743.12.41.6858069
    Visual Studio: 16.9.31205.134 (Visual Studio Professional 2019)
  Languages:
    Java: javac 15 - C:\Users\MeilickD\dev\jdk-15\bin\javac.EXE
  npmPackages:
    @react-native-community/cli: Not Found
    react: 17.0.1 => 17.0.1
    react-native: 0.64.0 => 0.64.0
    react-native-windows: ^0.64.0-0 => 0.64.8
  npmGlobalPackages:
    *react-native*: Not Found

## Steps To Reproduce
Provide a detailed list of steps that reproduce the issue.

1.  npx react-native init WebView64 --template react-native@^0.64.0
2.  cd WebView64 
3.  npx react-native-windows-init --overwrite --language cs
4.  npm install react-native-webview
5.  Change app.js to:
```
import React from 'react';
import type {Node} from 'react';
import {
  View,
} from 'react-native';
import {WebView} from 'react-native-webview'

const App: () => Node = () => {
  return (
    <View>
      <WebView source={{uri: 'https://reactnative.dev/'}} />
    </View>
  );
};

export default App;
```
6.  npx react-native run-windows

## Expected Results
Describe what you expected to happen.

I expect a react-native window containing a webview container showing the react native web site.

Instead, I get the "Invariant Violation: requireNativeComponent: "RCTWebView" was not found in the UIManager."  error.

I also tried linking with `npx react-native autolink-windows`.  No difference.

I tried building the C# app in Visual Studio, then running the app again with `npx react-native run-windows`.  No difference.

react-native-webview version is 11.4.5

## Snack, code example, screenshot, or link to a repository:
Please provide a Snack (https://snack.expo.io/), a link to a repository on GitHub, or provide a minimal code example that reproduces the problem.
You may provide a screenshot of the application if you think it is relevant to your bug report.
Here are some tips for providing a minimal example: https://stackoverflow.com/help/mcve

**Comment by dbm5858** [comment]
The problem does not seem to be related to a C# main project.  I also tried a C++ project, using the following steps, and it resulted in the same error.

1. npx react-native init WebView64Cpp --template react-native@^0.64.0
2. cd WebView64Cpp
3. npx react-native-windows-init --overwrite 
4. npx react-native run-windows
    The default app is displayed properly
5. npm install react-native-webview
6. Change App.js to the code included above
7. npx react-native run-windows
    This results in the same error cited above
8.  npx react-native autolink-windows
```
 √ Auto-linking...
Success: No auto-linking changes necessary. (2635ms)
```

**Comment by dbm5858** [comment]
I was able to manually link webview into a C# project.

1. npx react-native init WebView64 --template react-native@^0.64.0
2. cd WebView64
3. npx react-native-windows-init --overwrite --language cs
4. npx react-native run-windows (displays the default app)
5. npm install react-native-webview
6. Open studio
7. Add ..\node_modules\react-native-webview\windows\ReactNativeWebView\ReactNativeWebView.vcxproj to the solution
8. Add ..\node_modules\react-native-webview\windows\WebViewBridgeComponent\WebViewBridgeComponent.vcxproj to the solution
9. Add a reference to the ReactNativeWebView to the WebView64 project
10. Build the ReactNativeWebView project
11. Build the solution 
12. Change app.js as described above
13. npx react-native run-windows

The app now works as expected.

Seems like auto-linking is completely non-functional for react-native-webview in a Windows project.  But the component works, if you manually link it.

If I didn't explicitly build the ReactNativeWebView project (step 10), the solution wouldn't build.  It looks like even with manual linking, somehow the dependencies aren't right.

**Comment by asklar** [comment]
autolinking for webview was recently fixed (I know we were making changes up until yesterday...) @chiaramooney can comment on what version of webview should work

**Comment by chiaramooney** [comment]
Discovered the bug in 0.64 that is causing the issue. Fix on the way now.

**Comment by jonthysell** [comment]
> I just created a new RNW 0.64 C# project, and added react-native-webview 11.4.6 (released this morning). Steps as above.
> 
> Same error.
> 
> I'm willing to help debug this, but I am at a loss as to where to go from here.

We've found the issue. We fixed the issue in master in #7438 but in backporting to 0.64 (#7447) and 0.63 (#7452) there's a subtle typo that breaks it.

in master:

https://github.com/microsoft/react-native-windows/blob/471a89dc916d9a7098ac3892981b29c0c549d6f9/packages/%40react-native-windows/cli/src/runWindows/utils/autolink.ts#L437

in 0.64:

https://github.com/microsoft/react-native-windows/blob/061fc642d25fa40fb9a7e08a888a88bc10eabb40/packages/%40react-native-windows/cli/src/runWindows/utils/autolink.ts#L270

in 0.63:

https://github.com/microsoft/react-native-windows/blob/13741b68b5e230cd880d154aa63b6e414a514e9d/packages/%40react-native-windows/cli/src/runWindows/utils/autolink.ts#L255

@chiaramooney is looking at the fix.

**Comment by dbm5858** [comment]
That darned identity operator will get you every time!  ;-)

@jonthysell :  Thanks for the update.  I'll put this aside while the fix is in process.

**Comment by asklar** [comment]
Closing as this should be fixed in 0.63 upwards now

**Comment by dbm5858** [comment]
I've confirmed that auto-linking of react-native-webview is now working in an RNW 0.64 C# app.  My issues have been resolved.  Thanks folks!
